### PR TITLE
Feautre/#28

### PR DESCRIPTION
**Pull request by FirdavsAsadov**
category an result model is ready :)

### CODE REVIEW

File: src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Category.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestGorilla.Domain.Models
```
Comment (azam-amonov) [inline src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Category.cs:3]: Clean your code

File: src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Category.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+{
+    public class Category : Auditable
+    {
+        public string Name { get; set; }
+        public Category()
```
Comment (azam-amonov) [inline src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Category.cs:8]: Add here a line to split cosntructor and properties

File: src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Result.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```
Comment (azam-amonov) [inline src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Result.cs:2]: Clean your code from useless lines

File: src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Result.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    public class Result : Auditable
+    {
+        public long UserId { get; set; }
+        public long TestId { get; set; }
+        public long ExamId { get; set; }
+        public long CategoryId { get; set; }
+        public decimal TestResult { get; set; }
+        public decimal ExamResult { get; set;}
+        public bool IsDelete { get; set; }
+        public Result()
```
Comment (azam-amonov) [inline src/Training.TestGorilla.Clone.Backend/TestGorilla.Domain/Models/Result.cs:14]: Add line to split properties and constructor

**Review by azam-amonov — CHANGES_REQUESTED** [review]

**Review by azam-amonov — APPROVED** [review]

**Review by Jasurbek1223 — APPROVED** [review]

**Pull request merged**